addons/sys_indexer/fnc_auto_staticObjects.sqf

```diff
@@ -320,14 +320,14 @@ if (_mapBounds != 0) then {
 {
     private ["_array","_arrayActual","_result"];
     _array = _x select 0;
-    _arrayActual = missionNamespace getVariable _array;
+    _category = missionNamespace getVariable _array;
     // Window length set based on an example of cut off, 100 objects is probably accurate, 60 is conservative
     _windowLength = 60;
-    if (count _arrayActual >0) then {
+    if (count _category >0) then {
```

Comment:
if !(_category isEqualTo [])

is faster than count check.

Reply:
Resolved in next commit